### DISCUSSION TITLE
Number of scalars of stress is mismatch with number of points/cells

### DISCUSSION THREAD

**Discussion by willywah** [ansys/pytwin#164] [General]
Hi everyone, 

I'm new to working with PyAnsys, particularly PyTwin. I've created a simple simulation of a simply supported beam in Ansys Workbench and exported the static ROM to Twin Builder. The only input is the value of the distributed force applied to the structure.
![image](https://github.com/ansys/pytwin/assets/95516289/9fb72769-96c2-404d-8a12-39b55d7cbbde)

The system model has already been compiled and exported as twin files for both deformation and stress. I'm encountering an issue when using `twin_model.initialize_evaluation`. While it works fine for the deformation file, I get an error for the stress file.

> Number of scalars (3446) must match either the number of points (1723) or the number of cells (1723).

Can anyone explain what might be causing this error?

**Comment by chrpetre** [comment]
hello @WillyW28,
thank you for your interest in PyAnsys/PyTwin. In order to investigate the problem further, could you please share the python code used to get to this error, as well as the .twin file to reproduce if possible ?

**Comment by willywah** [comment]
Sure, please take a look at it here. 
[PyTwin_Stress.zip](https://github.com/user-attachments/files/16023202/PyTwin_Stress.zip)

It is a short code for twin model initialization. For deformation, all is working fine (at least for twin initialization).

**Comment by chrpetre** [comment]
hi @WillyW28 

thanks for the files. It helped identifying a small issue that has been fixed through the following PR : 
https://github.com/ansys/pytwin/pull/165

You should be able to re-install PyTwin from the development branch : pip install git+https://github.com/pyansys/pytwin.git@main
and test/confirm it solves the problem identified, thanks!

**Reply by willywah** [reply]
hi @chrpetre, 

this case is already working fine. Thanks

**Reply by ctrwtskm** [reply]
hi @chrpetre following this up, I encountered the exact same problem again (I use the similar twin model with @willywah used) using PyTwin 0.7.0, but when I rolled back the version into 0.6.0 the model can be initialized well.
Is the issue reappeared in the current version?

**Reply by chrpetre** [reply]
hello @ctrwtskm, can you please share the model and python code used on your end to reproduce this ?

**Reply by ctrwtskm** [reply]
hi @chrpetre please find the zip file below, I also added the error message screenshot
![image](https://github.com/user-attachments/assets/12c89575-180f-497d-8862-5dba233990d2)
[PyTwin_Stress.zip](https://github.com/user-attachments/files/17616696/PyTwin_Stress.zip)

**Reply by chrpetre** [reply]
@ctrwtskm please note that the issue reported here had been fixed after PyTwin 0.7.0, and therefore is currently available from the main branch only (a new release is expected in the coming weeks that should include this update as well) 

you can install the main branch using : pip install git+https://github.com/pyansys/pytwin.git@main
(currently at version 0.8.dev0 - with this version, I confirm the reported issue does not appear anymore)

**Reply by ctrwtskm** [reply]
hi @chrpetre, I already re-installed the PyTwin as per your instruction, and now all good.
thank you!
